lte/gateway/c/core/oai/tasks/sgw_s8/sgw_s8_state_manager.cpp

```diff
@@ -124,7 +124,6 @@ void SgwStateManager::free_state() {
                    "temporary_create_session_procedure_id_map ");
     }
   }
-  s8_state_teid_map.map = nullptr;
```

Comment:
Why is this removed?

Reply:
This is removed because map is set to nullptr within destroy_map() itself. This change is done while addressing the comment, https://github.com/magma/magma/pull/14006#discussion_r1007844896